internal/oci/container.go

```diff
@@ -79,6 +80,11 @@ type ContainerState struct {
 	ExitCode  *int32    `json:"exitCode,omitempty"`
 	OOMKilled bool      `json:"oomKilled,omitempty"`
 	Error     string    `json:"error,omitempty"`
+	InitPid   int       `json:"initPid,omitempty"`
+	// The unix start time of the container's init PID.
+	// This is used to track whether the PID we have stored
+	// is the same as the corresponding PID on the host.
+	InitStartTime string `json:"initStartTime,omitempty"`
```

Comment:
It never occurred to be before, but why are we using a `string` rather than `int64` here. Using an integer would be easier (no conversions), faster (to compare two ints rather than two strings), and it will look almost the same in json anyway.

Reply:
fixed!